introduction/project_setup.md

```diff
@@ -1,20 +1,59 @@
-## プロジェクトのセットアップ {#project-setup}
+# プロジェクトのセットアップ {#project-setup}
 
-説明を始める前に、どのようにVisual StudioまたはXamarin等でプロジェクトをセットアップするかを示す。
+説明を始める前に、プロジェクトをどのように構成するべきかを説明する。
 
-1. 新規プロジェクトを作成する（.NET 4.5以上）
+1. 新規プロジェクトを作成する（.NET 4.5.2以上）
 2. ソリューションエクスプローラーの「参照」で右クリックし、「NuGet パッケージの管理」をクリック
-3. 「**NBitcoin**」を検索しインストールする。
-   ![](../assets/nuget_jp.png)  
+3. 「**NBitcoin**」を検索しインストールする
+![nuget](../assets/nuget_jp.png)
 
-> **ヒント:** MacやLinuxでも、NBitcoin.Monoではなく、NBitcoinを使うこと。
+> **ヒント:** もしMacやLinuxを使っている場合、.NET coreをインストールした上で、NBitcoinをインストールしましょう。以前のバージョンではNBitcoin.Monoが推奨されていましたが、現在はアップデートされていません（翻訳者注）。
```

Comment:
I think he is saying the other way around.
MACとLinuxの場合で、NBitcoin.Monoでなく、NBitcoinライブラリーに依存していると、いくつかのクラスがないという問題がおきる。

Reply:
@hypergori I just now understood what Mono is and NBitcoin.Mono is.

Mono is an open source implementation of .NET framework. `NBitcoin.Mono` is also something that enables Mac to use `NBitcoin`. I found the last updated date of `NBitcoin.Mono` is `8/10/2016`, which caused me to wonder I might not be able to use the latest features of `NBitcoin`. But it is wrong because `NBitcoin.Mono` is just for translating `NBitcoin`, developed with C# for Mac.